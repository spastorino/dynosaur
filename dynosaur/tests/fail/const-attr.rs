#[dynosaur::dynosaur(DynFoo)]
trait Foo {
    const BAR: i32;

    fn foo(&self) -> impl Send;
}

fn main() {}
