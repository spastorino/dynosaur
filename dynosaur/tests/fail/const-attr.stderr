error: consts make the trait not dyn compatible
 --> tests/fail/const-attr.rs:3:5
  |
3 |     const BAR: i32;
  |     ^^^^^^^^^^^^^^^

error: aborting due to 1 previous error

